unlock-app/src/components/creator/CreatorLockForm.js

```diff
@@ -195,15 +291,30 @@ const LockLabelUnlimited = styled(LockLabel)`
 `
 
 const FormLockRow = styled(LockRow)`
-  ${LockRowGrid} input[type='text'] {
-    background-color: var(--lightgrey);
-    border: 0;
-    padding: 5px;
-    font-family: 'IBM Plex Sans', sans-serif;
-    font-size: 13px;
+  input[type='number'] {
+    -moz-appearance: textfield;
```

Comment:
Meh. Are you sure this is a good idea to use prefixed-CSS stuff? I thought this would be handled by our build tools?

Reply:
in this case, the appearance thing is only non-standard, and this is what was recommended to remove the up-down spinner for numbers on all browsers. Perhaps in a year or two, the browsers will standardize something, but for now, we are in 2010 when it comes to number input styling :)